action.yml

```diff
@@ -21,8 +21,9 @@ inputs:
     description: 'The tag name of the release to generate notes for.'
     required: true
   chapters:
-    description: 'JSON string defining chapters and corresponding labels for categorization.'
+    description: 'An YAML array defining chapters and corresponding labels for categorization.'
```

Comment:
In this phase the input is not an YAML array. Update that comment that it is reflecting an array, but it is in a string format plaese.